packages/mobile/src/components/help-callout/HelpCallout.tsx

```diff
@@ -22,26 +24,23 @@ const useStyles = makeStyles(({ spacing, palette }) => ({
   text: {
     flex: 1,
     flexWrap: 'wrap'
-  },
-  questionIcon: {
-    marginRight: spacing(4),
-    width: spacing(5),
-    height: spacing(5)
   }
 }))
 
 type HelpCalloutProps = {
   content: ReactNode
   style?: ViewStyle
+  icon?: ComponentType<SvgProps>
 }
 
-export const HelpCallout = ({ content, style }: HelpCalloutProps) => {
+export const HelpCallout = (props: HelpCalloutProps) => {
```

Comment:
Worth noting this is the `Hint` component. We should rename (web and mobile) cc: @dylanjeffers RE: Harmony

Reply:
Can add a comment, perhaps

// TODO(harmony): This is `Hint`